src/middlewared/middlewared/utils/country_codes.py

```diff
@@ -0,0 +1,10 @@
+from functools import cache
+from json import load
+
+__all__ = ("get_iso_3166_2_country_codes",)
+
+
+@cache
+def get_iso_3166_2_country_codes() -> dict[str, str]:
+    with open("/usr/share/iso-codes/json/iso_3166-1.json") as f:
```

Comment:
Is it a problem that this is `iso_3166-1` and endpoint says `3166-2`?

Reply:
I'm returning alpha_2 which is the 2 digit country code